pkg/morph/client/notary.go

```diff
@@ -263,18 +266,18 @@ func (c *Client) UpdateNeoFSAlphabetList(list keys.PublicKeys) error {
 // it fallbacks to a simple `Invoke()`.
 //
 // This function must be invoked with notary enabled otherwise it throws panic.
-func (c *Client) NotaryInvoke(contract util.Uint160, fee fixedn.Fixed8, method string, args ...interface{}) error {
+func (c *Client) NotaryInvoke(contract util.Uint160, fee fixedn.Fixed8, nonce uint32, method string, args ...interface{}) error {
```

Comment:
Isn't it time to structure the parameters in `NotaryInvokePrm`?

Reply:
i think it is a good idea, but also I think that it would be good to keep `Invoke` and `NotaryInvoke` funcs in sync so add `Prm` structs for both of them or even it should be the same struct

/cc @alexvanin

Reply:
Then you can define `InvokePrm`, and inherit it in `NotaryInvokePrm`. I think single type is not a good idea.